package.json

```diff
@@ -15,7 +15,8 @@
 		"lint-staged": "lint-staged",
 		"format": "prettier --cache --write .",
 		"preinstall": "npx only-allow npm",
-		"prepare": "husky"
+		"prepare": "husky",
+		"hestia": "cd src && ./hst_autocompile.sh --hestia --install '~localsrc'"
```

Comment:
This feels like an unrelated change. If we add this it should be included in the development docs etc. it's probably better in a separate PR if you're keen on this.

Reply:
OK i will remove it for now